hal_st/synchronous_stm32fxxx/SynchronousAesStm.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef HAL_SYNCHRONOUS_AES_STM_HPP
+#define HAL_SYNCHRONOUS_AES_STM_HPP
+
+#include DEVICE_HEADER
+#include "generated/stm32fxxx/PeripheralTable.hpp"
+#include "infra/util/ByteRange.hpp"
+
+#if defined(HAS_PERIPHERAL_AES)
+
+namespace hal
+{
+    class SynchronousAesEcbStm
+    {
+    public:
+        enum class OperationMode : uint8_t
+        {
+            encrypt,
+            decrypt,
+        };
+
+        struct Config
+        {
+            bool enableReverse = false;
```

Comment:
What do enableReverse and enableSwap do? If there's no better name to describe their function, at a minimum I would rename these to reverse and swap

Reply:
Fixed